gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/AllergyIntoleranceMapperTest.java

```diff
@@ -51,14 +52,17 @@ public class AllergyIntoleranceMapperTest {
     private static final String CODING_DISPLAY_1 = "Ischaemic heart disease";
     private static final String CODING_DISPLAY_2 = "H/O: aspirin allergy";
     private static final String CODING_DISPLAY_3 = "H/O: drug allergy";
+    private static final String CODING_DISPLAY_4 = "Coconut oil";
     private static final String PATIENT_ID = "9A5D5A78-1F63-434C-9637-1D7E7843341B";
     private static final String META_PROFILE = "https://fhir.nhs"
         + ".uk/STU3/StructureDefinition/CareConnect-GPC-AllergyIntolerance-1";
     private static final String IDENTIFIER_SYSTEM = "https://PSSAdaptor/TESTPRACTISECODE";
-    private static final String NOTE_TEXT = "Reason Ended: Patient reports no subsequent recurrence on same medication Status:"
+    private static final String EPISODICITY_WITH_ORIGINAL_TEXT_NOTE_TEXT = "Episodicity : Ongoing, Review";
```

Comment:
In the ticket, the [suggested text](https://gpitbjss.atlassian.net/browse/NIAD-2869?focusedCommentId=83432) was "Episodicity : Review".

Reply:
In which case we need to review this, as I read the ticket that we should be including both, if we want it to just show the originalText when it is present and lose the information relating to the Snomed Code.  Ongoing is an episodicity, where as review is the text manually entered.  It would seem to make sense to include both so no information is lost?